8/Api/src/ClusterClientHostedService.cs

```diff
@@ -102,30 +99,5 @@ public async Task StopAsync(CancellationToken cancellationToken)
                     "Error while gracefully disconnecting from Orleans cluster. Will ignore and continue to shutdown.");
             }
         }
-
-        private static IPAddress GetLocalIpAddress()
-        {
-            var networkInterfaces = NetworkInterface.GetAllNetworkInterfaces();
-            foreach (var network in networkInterfaces)
-            {
-                if (network.OperationalStatus != OperationalStatus.Up)
-                    continue;
-
-                var properties = network.GetIPProperties();
-                if (properties.GatewayAddresses.Count == 0)
-                    continue;
-
-                foreach (var address in properties.UnicastAddresses)
-                {
-                    if (address.Address.AddressFamily == AddressFamily.InterNetwork &&
-                        !IPAddress.IsLoopback(address.Address))
-                    {
-                        return address.Address;
-                    }
-                }
-            }
-
-            return null;
-        }
```

Comment:
This is not used and will not be needed as I'd like us to default to dynamodb clustering exclusively.